templates/blockonomics_checkout.php

```diff
@@ -95,35 +114,27 @@
             <h4><i class="material-icons bnomics-alt-icon">error</i></h4>
             <?= __('Unable to render correctly, Note to Administrator: Please enable lite mode in Blockonomics plugin.', 'blockonomics-bitcoin-payments') ?>
           </div>
-          <?php if (get_option('blockonomics_altcoins')) : ?>
-          <div class="bnomics-altcoin-pane" ng-style="{'border-left': (altcoin_waiting)?'none':''}" ng-hide="show_altcoin != 1">
-            <div class="bnomics-altcoin-bg">
-                <div class="bnomics-altcoin-bg-color" ng-hide="altcoin_waiting" ng-cloak>
-                 <div class="bnomics-altcoin-info-wrapper">
-                  <span class="bnomics-altcoin-info" ><?=__('Select your preferred <strong>Altcoin</strong> then click on the button below.', 'blockonomics-bitcoin-payments')?></span>
-                 </div>
-                 </br>
-                 <!-- Coin Select -->
-                 <div class="bnomics-address">
-                   <select ng-model="altcoinselect" ng-options="x for (x, y) in altcoins" ng-init="altcoinselect='Ethereum'"></select>
-                 </div>
-                 <div class="bnomics-altcoin-button-wrapper">
-                  <a ng-click="pay_altcoins()" href=""><button><i class="cf" ng-hide="altcoinselect!='Ethereum'" ng-class={'cf-eth':'{{altcoinselect}}'!=''} ></i><i class="cf" ng-hide="altcoinselect!='Litecoin'" ng-class={'cf-ltc':'{{altcoinselect}}'!=''} ></i> <?=__('Pay with', 'blockonomics-bitcoin-payments')?> {{altcoinselect}}</button></a>
-                 </div>
-                </div>
-            </div>
-          </div>
-          <?php endif ?>
-
         </div>
       </div>
+    </div>
+      <!-- Address Error -->
+      <div class="bnomics-display-error" style="text-align: center;" ng-show="btcaddresserror">
```

Comment:
Two different designs on error style is not needed. One div is enough and only change which error text is displayed